R-package/R/lgb.train.R

```diff
@@ -12,6 +12,8 @@
 #' @param categorical_feature list of str or int
 #'        type int represents index,
 #'        type str represents feature names
+#' @param callbacks list of callback functions
+#'        List of callback functions that are applied at each iteration.
```

Comment:
how is this change related?

Reply:
I find the doc of this parameter is misning, compared with lgb.cv.